### PR TITLE
Profile edit

### PR DESCRIPTION
**Pull request by JasmienVC**
image

Edit profile directly in showpage (we should replace that with js)

### CODE REVIEW

File: app/views/profiles/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for(@profile) do |f| %>
```
Comment (rubenvoss) [inline app/views/profiles/_form.html.erb:1]: pls wrap this in a container and add the red button to f.submit

File: app/views/profiles/new.html.erb [hunk 2]
```diff
@@ -1,8 +1,4 @@
-<h1>Make a profile</h1>
-
-<%= simple_form_for(@profile) do |f| %>
-  <%= f.input :name %>
-  <%= f.input :bio %>
-  <%= f.association :crag %>
-  <%= f.submit %>
-<% end %>
+<div class="container">
```
Comment (rubenvoss) [inline app/views/profiles/new.html.erb:1]: yesss! partials!! :D

**Review by rubenvoss — CHANGES_REQUESTED** [review]

File: app/views/profiles/_form.html.erb (hunk 1 above)
Comment (JasmienVC) [reply app/views/profiles/_form.html.erb:1]: added button in partial

**Comment by rubenvoss** [comment]
nice!

**Pull request merged**